engine/runtime/src/main/java/org/enso/interpreter/runtime/error/DataflowError.java

```diff
@@ -49,9 +42,11 @@ public final class DataflowError extends AbstractTruffleException {
    * @param location the node in which the error was created
    * @return a new dataflow error
    */
-  public static DataflowError withoutTrace(Object payload, Node location) {
+  public static DataflowError withoutTrace(State state, Object payload, Node location) {
```

Comment:
Should we rename this constructor?

It is not really `withoutTrace`, more like `withDefaultTrace` or something like that.

I'd also like to update the comment to mention that it has the default location (top elem) or full stack depending on the Context.